framework/src/seclink/seclink.c

```diff
@@ -69,20 +73,6 @@
 		}													 \
 	} while (0)
 
-#define SL_SET_MEMPOOL(hdata)                                 \
```

Comment:
Any reasons to remove mempool?
When we modify structure or architecture, we need to explain why.

Reply:
Sorry for missing. We originally had a plan to use seclink not to be initialized and uninitialized during the run-time, whenever security API is called by user. That means, if it is initialized once as a daemon, it has no reason to be uninitialized.
However, considering the structure of security API, it has no reason to run seclink as a daemon,  
so user now has to pass a (user-space-defined) buffer to seclink and then, seclink passes the buffer pointer to the hal, which will be filled with a requested data. Thanks for the comment.